com.cuperino.qprompt.appdata.xml

```diff
@@ -27,6 +28,11 @@
       <caption>Countdown on stand-by</caption>
     </screenshot>
   </screenshots>
+  <categories>
+    <category>Qt</category>
```

Comment:
Qt category should probably be removed since it's non-standard and QPrompt despide being made with Qt is not a Qt development tool. (I'll merge this and then remove the line. The rest is good.)